com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -4628,7 +4628,7 @@ private static string ConvertToMayaCompatibleName(string name)
         {
             string newName = RemoveDiacritics (name);
```

Comment:
I think the empty check should be before this call to be safe

Reply:
If the name is null or empty, should it exit or reassign something to the name? (Like `InvalidCharReplacement` or a default name maybe)

Reply:
If possible try to assign `InvalidCharReplacement`